src/docs/wasm-shipping-checklist.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Checklist for staging and shipping of WebAssembly features'
+description: 'This document provides checklists of engineering requirements on when to stage and ship a WebAssembly feature in V8.'
+---
+This document provides checklists of engineering requirements for staging and shipping WebAssembly features in V8. These checklists are meant
+as a guideline and may not be applicable to all features. The actual launch
```

Comment:
Nit: these two line breaks look odd, we generally don't break lines on limits.

Reply:
+1. Line breaks are turned into hard `<br>`s, which would look bad on various viewport widths. On v8.dev, only use line breaks when you need one semantically.

Reply:
Done. Could the linter check for that?

Reply:
Hmm, I'm not sure. I don't think the linter has any way of knowing (for sure) whether the linebreak you added was intentional or not.